avm/res/db-for-my-sql/flexible-server/tests/e2e/private/main.test.bicep

```diff
@@ -56,8 +56,11 @@ module testDeployment '../../../main.bicep' = [
     scope: resourceGroup
     name: '${uniqueString(deployment().name, enforcedLocation)}-test-${serviceShort}-${iteration}'
     params: {
-      name: '${namePrefix}${serviceShort}001'
+      name: 'mysql${namePrefix}${serviceShort}001'
```

Comment:
Is the addition of mysql at the front of the name intentional?

Reply:
I added that for troubleshooting why it wasn't creating. I can remove.